indico/migrations/versions/20200319_1605_56a26a721717_.py

```diff
@@ -0,0 +1,29 @@
+"""Add slides as EditingType
+
+Revision ID: 56a26a721717
+Revises: 18a1088f1ea8
+Create Date: 2020-03-19 16:05:28.021616
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '56a26a721717'
+down_revision = '18a1088f1ea8'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    op.execute('''
+        ALTER TABLE "event_editing"."editables" DROP CONSTRAINT "ck_editables_valid_enum_type";
+        ALTER TABLE "event_editing"."editables" ADD CONSTRAINT "ck_editables_valid_enum_type" CHECK ((type = ANY (ARRAY[1, 2, 3])));
```

Comment:
Maybe wrap this like to make it more readable (it is a bit too long.

```sql
         ALTER TABLE event_editing.editables
         ADD CONSTRAINT "ck_editables_valid_enum_type"
         CHECK ((type = ANY (ARRAY[1, 2, 3])));
```

Reply:
I really wouldn't bother with wrapping those raw SQL statements. IMHO it doesn't even help readability (not that you really *want* to read them).

Reply:
I do read them :P and for example with GH view it is hard to see the whole statement

Reply:
fair enough, i usually just use dbdiff to check that the revision works, and is in sync with the models (which are much more readable)..

anyway, i still think 1 statement per line for *bulk* raw sql (especially in cases where you have maybe 20+ lines of generated sql) is easier.

fwiw, in this particular case (one constraint) using the standard alembic operations to create/drop constraints wouldn't be so bad (i didn't look at how much sql we have here when writing my last comment about it being a waste of time)

Reply:
I personally didn't find those particular two lines unreadable. I think it's kind of a matter of preference in this case as I don't prefer so much wrapping as posted in the comment.

Reply:
To me, as a reviewer, it was a bit unreadable but in the end I am not against leaving it as it is.